src/tests/Dictionary/DictionaryModal.test.jsx

```diff
@@ -206,4 +206,24 @@ describe('Test suite for dictionary modal', () => {
     };
     wrapper.find('#copy_dictionary').simulate('change', item);
   });
+
+  it('it should disable button if there is no error', () => {
```

Comment:
Is the ticket about disabling the button or reenabling it?

Reply:
it is about reenabling it if an error occurs and disabling it otherwise.